### ISSUE TITLE
When livekit/sip generates the BYE, it comes without the FROM tag in the header

### ISSUE THREAD

**Issue by Otoru** [livekit/sip#177]
Good afternoon everyone,

Title is very descriptive. I am attaching an example capture when using the agent to generate a BYE on the livekit side, it occurs, but it is sent without the From Tag in the header.

[example.zip](https://github.com/user-attachments/files/17017135/example.zip)

Example BYE:
```
Via: SIP/2.0/UDP 132.226.255.138:5060;branch=z9hG4bK.iOIHmeaPsaQGs3yJ;alias
Max-Forwards: 70
Call-ID: 624b3702-eef3-123d-14a7-0200170133f2
CSeq: 88699946 BYE
Content-Length: 0
From: <sip:554833328563@162.213.197.186:5060>
To: "48991163842" <sip:48991163842@132.226.255.138>;tag=NZN4y3tDSevyH
```

I'm using livekit/sip v0.7 with docker.

**Comment by ftsef** [comment]
already mentioned here
https://github.com/livekit/sip/issues/112 and https://github.com/livekit/sip/pull/118

I think a fix is not available in 0.7 since the root cause has been found last month and 0.7 is older.
If I have time I'll test the latest master.

**Comment by dennwc** [comment]
That's true, we should cut a new release. Master definitely has a fix for this one.

**Comment by davidzhao** [comment]
closing this issue since it's fixed in #112. it'll be addressed in the next release
